templates/base/head.tmpl

```diff
@@ -25,14 +25,17 @@
 	<meta name="go-import" content="{{.GoGetImport}} git {{.CloneLink.HTTPS}}">
 	<meta name="go-source" content="{{.GoGetImport}} _ {{.GoDocDirectory}} {{.GoDocFile}}">
 {{end}}
+	{{/* here we have valid go template syntax, but eslint doesn't like it and reports "error  Parsing error: Unexpected token {" */}}
 	<script>
+		<!-- /* eslint-disable */ -->
 		window.config = {
 			AppVer: '{{AppVer}}',
 			AppSubUrl: '{{AppSubUrl}}',
 			AssetUrlPrefix: '{{AssetUrlPrefix}}',
 			CustomEmojis: {{CustomEmojis}},
 			UseServiceWorker: {{UseServiceWorker}},
 			csrf: '{{.CsrfToken}}',
+			PageData: {{ .PageData }},  // to store page related data, eg: pass variables to javascript module
```

Comment:
Maybe better use `pageData` as that would more conform with JS naming

Reply:
I think PageData is better because:

1. Most other variables look like this
2. It indicates that this is not an ordinary variable for local or ad-hoc usage, it has special meaning and should not be modified manually.

Reply:
I agree with @wxiaoguang, the other vars above use this naming scheme, as it aligns with how the naming scheme used internally.

Although, as a note as feedback for this specific line, perhaps putting the comment in a new line right before the code is better as then there is not a long line that is code & comment together.

Reply:
Maybe we can remove the frontend comment (otherwise is will be rendered to the HTML)

There is already a comment about `PageData`, https://github.com/go-gitea/gitea/pull/16770/files#diff-fa07e296d614e796de67f7ae9268a0660b89c09b6e5f6373b4ca7bf76a20dec7R54 , it seems enough

Reply:
I know it does differ from other used here but if we are starting new fronted plan I would prefer we would start using correct JS naming everywhere.
Others should also be renamed later :)

Reply:
Then we can rename them together, otherwise the mixed naming seems very strange 😂

I prefer to stick to the existing style, except refactor all together.

Reply:
Refactor should then happen in different PR